scripts/remove-import-groups.sh

```diff
@@ -6,8 +6,8 @@
 temp=$(mktemp)
 for each in $(fd '.*\.hs$')
 do
-    tac "$each" > "$temp"
-    sed -i '/^import/,/^module/{/^[[:space:]]*$/d}' -i "$temp"
+    ( cat "$each"; echo ) | tac - > "$temp"
```

Comment:
I'm not sure I understand this construct. How does that `echo` with no arguments work?

Reply:
It prints a blank line. Basically, this ensures that the file always ends in a newline, and relies on Fourmolu to clean it up if we accidentally add too many.